composer.json

```diff
@@ -161,9 +159,14 @@
             "web/modules/contrib/{$name}": ["type:drupal-module"],
             "web/profiles/contrib/{$name}": ["type:drupal-profile"],
             "web/themes/contrib/{$name}": ["type:drupal-theme"],
+            "drush/contrib/{$name}": ["type:drupal-drush"],
             "drush/Commands/{$name}": ["type:drupal-drush"]
```

Comment:
Mind deleting the `drush/Commands` entry? Don't want duplicate `type`s showing up.